src/tv2-common/migrations/index.ts

```diff
@@ -125,6 +129,61 @@ export function AddGraphicToGFXTable(
 	}
 }
 
+export function mapGFXTemplateToDesignTemplate(versionStr: string, studio: string, from: string, to: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.mapGFXTemplateToDesignTemplate.${from}.${studio}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[] | undefined
+
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[] | undefined
+
+			if (!gfxTemplates || !gfxTemplates.length) {
+				return false
+			}
+
+			if (designTemplates && designTemplates.length) {
+				return false
+			}
+
+			return gfxTemplates.some(g => g.IsDesign)
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const gfxTemplates = (context.getBaseConfig(from) as unknown) as TableConfigItemGFXTemplates[]
+			const designTemplates = (context.getBaseConfig(to) as unknown) as TableConfigItemGFXDesignTemplates[]
+
+			gfxTemplates.filter(g => g.IsDesign).map(g => designTemplates.push(g))
```

Comment:
I think this way we're going to have some unwanted properties on these rows. I'm not sure if they will be deleted when editing the table. Might be worth looking into.

Reply:
A new implementation has been made for this in [73a488c](https://github.com/tv2/sofie-blueprints-inews/pull/149/commits/73a488c30f009df70d38e047371d4b47fdf749ec)